docs/api/build_proto.md

```diff
@@ -0,0 +1,346 @@
+# Building gRPC proto
+
+This page shows how to build gRPC proto file for calling API to your Vald Cluster.
+
+<div class="information">
+Vald provides [the official client libraries](../user-guides/sdks.md), if you can use one of SDKs we recommend to use it.
```

Comment:
I don't know why the link cannot be shown, can you please try to fix it?
```suggestion
Vald provides [the official client libraries](../user-guides/sdks.md), if you can use one of SDKs we recommend to use it.
```

Reply:
I think it may be because this sentence is wrapped by `div`.